Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -509,35 +509,42 @@ extension SwiftLanguageServer {
   public func hover(_ req: HoverRequest) async throws -> HoverResponse? {
     let uri = req.textDocument.uri
     let position = req.position
-    guard let cursorInfo = try await cursorInfo(uri, position..<position) else {
-      return nil
-    }
+    // LSP's hover can't display multiple results. Don't display any results if the cursor info response is
```

Comment:
Did you change your mind here and concat instead of failing if there were multiple 😅?

Reply:
Yes. I think only showing the primary result when it could be ambiguous can be actively misleading.

Reply:
Totally fine, in case it wasn't clear my comment was about how the comment is old.

```suggestion
    // LSP's hover can't display multiple results. Concatenate any alternate results together.
```

Reply:
Ah, I didn’t see that when going through the review.